### ISSUE TITLE
torchcomplex.nn.functional supports the mseLoss ? 

### ISSUE THREAD

**Issue by KarroyWang001** [soumickmj/pytorch-complex#1]
Thanks very much for your torchcomplex. I have only one question, what if my loss is to minimize the mse loss between two complex vector? That is, I want to use nn.MSELoss, however, which seems not exit now.

**Comment by soumickmj** [comment]
Hi @KarroyWang001 
Sorry for getting back to you so late. Not sure if this question is stil valid.
For the loss, you can use nn.L1Loss from PyTorch.
I haven't implemented any of our own loss functions, rather using from PyTorch. Currently, PyTorch only supports complex-valued L1 loss.
